packages/amplication-server/src/core/auth/auth.service.ts

```diff
@@ -188,6 +213,25 @@ export class AuthService {
       throw new AmplicationError("Invalid password");
     }
 
+    this.kafkaProducerService
+      .emitMessage(
+        EnvironmentVariables.instance.get(Env.USER_ACTION_TOPIC, true),
+        {
```

Comment:
Try to add the type you need to use to avoid missing mandatory parameters (as `userId` in the key)
```suggestion
        <UserAction>{
```